core/deployment/src/main/java/io/quarkiverse/jberet/deployment/JBeretProcessor.java

```diff
@@ -99,28 +110,37 @@ public class JBeretProcessor {
     private static final DotName JOB = DotName.createSimple(Job.class);
 
     @BuildStep
-    public void registerExtension(BuildProducer<FeatureBuildItem> feature, BuildProducer<CapabilityBuildItem> capability) {
+    public void registerExtension(BuildProducer<FeatureBuildItem> feature,
```

Comment:
It seems that in this class, there are a bunch of changes that are only code style changes and not related to the actual feature. Any chance we can revert these and keep only the relevant changes?